test/rulesets/LinearAlgebra/structured.jl

```diff
@@ -87,6 +87,10 @@
             end
         end
     end
+    @testset "sqrt(::Diagonal)" begin
+        test_rrule(sqrt, Diagonal(rand(3)))
```

Comment:
If you're encountering stochastic issues with FD with the test below I think you'll still be able to hit them here because of the implicit call here to `rand_tangent`.

Reply:
it's weird, this one never errored - i'd be in favour of merging once CI passes so that Tom can wrap up (and solving this if it ever breaks)